### PR TITLE
[IOTDB-2413] Fix file handler not released bug 

### PR DESCRIPTION
**Pull request by JackieTien97**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java [hunk 1]
```diff
@@ -53,30 +53,9 @@
 import org.apache.iotdb.db.qp.logical.crud.UDAFQueryOperator;
 import org.apache.iotdb.db.qp.logical.crud.UDTFQueryOperator;
 import org.apache.iotdb.db.qp.logical.crud.WhereComponent;
-import org.apache.iotdb.db.qp.logical.sys.ActivateTemplateOperator;
-import org.apache.iotdb.db.qp.logical.sys.AlterTimeSeriesOperator;
+import org.apache.iotdb.db.qp.logical.sys.*;
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:56]: Good job.
this does not import *

File: server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java [hunk 2]
```diff
@@ -1998,6 +1977,12 @@ public Operator visitShowLockInfo(IoTDBSqlParser.ShowLockInfoContext ctx) {
     }
   }
 
+  // Show Query Resource
+
+  @Override
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:1982]: Reduce unnecessary blank lines

File: server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java [hunk 3]
```diff
@@ -192,6 +192,8 @@ public PhysicalPlan generatePhysicalPlan(PhysicalGenerator generator)
     UNSET_TEMPLATE,
     PRUNE_TEMPLATE,
     APPEND_TEMPLATE,
-    DROP_TEMPLATE
+    DROP_TEMPLATE,
+
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java:196]: Reduce unnecessary blank lines

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java (hunk 2 above)
Comment (HTHou) [reply server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java:1982]: Is this comment for IDEA index?

**Review by qiaojialin — APPROVED** [review]

File: server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java (hunk 3 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java:196]: there is another type Operator behind, so a blank is here to separate them~

**Pull request merged**